### ISSUE TITLE
Individual levels for multi handler

### ISSUE THREAD

**Issue by claytoncasey01** [apex/log#59]
It would be nice when using the multi handler if we could set the individual logging level for each handler. For example in my application I only want info to be logged to the cli and Warn, Error, and Fatal to be written out as json to a file.

Based on an initial look this might require a lot of changes since Level is currently part of the Logger struct.

**Comment by tj** [comment]
Agreed, I'd like if Logger/Entry had parity, might have to be a v2 thing
